grails-app/services/com/mini/asaas/customer/CustomerService.groovy

```diff
@@ -26,4 +26,23 @@ class CustomerService {
 
         return customer
     }
+
+    public Customer update(Long customerId, params){
+        Customer customer = Customer.get(customerId)
+
+        customer.name = params.name
+        customer.email = params.email
+        customer.phone = params.phone
+        customer.postalCode = params.postalCode
+        customer.address = params.address
+        customer.addressNumber = params.addressNumber
+        customer.addressComplement = params.addressComplement
```

Comment:
Se não for usar o adapter, aqui tu teria que fazer a mesma coisa que no adapter pra garantir que se a pessoa não passar nada pra complemento a aplicação salva como null